src/shared/components/cnSegments/CNSegmentsDownloader.tsx

```diff
@@ -27,7 +27,6 @@ export default class CNSegmentsDownloader extends React.Component<CNSegmentsDown
     };
 
     @observable downloading = false;
-    private downloadFileContent: string|undefined;
```

Comment:
removing this because caching download file content is problematic when the promise is updated due to filters